### ISSUE TITLE
Speech to Text not working

### ISSUE THREAD

**Issue by zixixr** [lucoiso/UEAzSpeech#187]
It's weird that I didn't change anything, but the Speech to Text suddenly doesn't work, the Text to Speech still works, and the api key is still working.

I've tried earlier version and latest version of plugin, and tried different PC, all don't work. I have turned on the Azure logs, no warnings or errors.

**Comment by debellica** [comment]
Hmm... this might be related [Speech.extension.codec.dll](https://github.com/lucoiso/UEAzSpeech/issues/194) issue perhaps. Could it be that you have something installed on your PC that isn't translated to us in the binaries, @lucoiso?

**Comment by lucoiso** [comment]
I'm still searching for what is causing this. Can you send me the Azure SDK Logs? :0
STT is still working well here in both Editor and packaged projects.

Found some recent issues related to TTS:
1. https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/1923
2. https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/1921
3. https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/1919
4. https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/1911

---

Enable Azure SDK Logs in: Project Settings -> Plugins -> AzSpeech -> Information -> Enable Azure SDK Logs
Logs are generated in: PROJECT_DIRECTORY/Saved/Logs/AzSpeech

**Comment by debellica** [comment]
Definitely try clearing the cache @zixixr, I have resolved my issue pertaining to this by deleting Saved/Intermediate/Binaries etc from project and plugins. 

@lucoiso from what i could see the internal logs were all fine - no warnings or errors (however there's possibly other indicators I am not seeing). I have deleted those logs unfortunately in the countless rebuilds trying to get it to work - and eventually succeeding. Next time something like this happens i'll drop the logs into the corresponding thread as it occurs (after first seeing if its not a fault of my own lol).
